node/lib/cmd/pull.js

```diff
@@ -97,8 +97,10 @@ exports.executeableSubcommand = co.wrap(function *(args) {
         throw new UserError("Non-rebase pull not supported.");
     }
 
+    // TODO: move the following code into `util/push.js` and add test
+
     const branch = yield repo.getCurrentBranch();
-    const tracking = yield GitUtil.getTrackingInfo(branch);
+    const tracking = (yield GitUtil.getTrackingInfo(branch)) || {};
```

Comment:
Does this mean that getTrackingInfo returns the wrong thing?  That is, should it return {} instead of null?